x/superfluid/keeper/hooks.go

```diff
@@ -88,3 +88,9 @@ func (h Hooks) BeforeValidatorSlashed(ctx sdk.Context, valAddr sdk.ValAddress, i
 	}
 	h.k.SlashLockupsForValidatorSlash(ctx, valAddr, infractionHeight, slashFactor)
 }
+func (h Hooks) AfterValidatorSlashed(ctx sdk.Context, valAddr sdk.ValAddress, infractionHeight int64, slashFactor sdk.Dec, effectiveSlashFactor sdk.Dec) {
+	if slashFactor.IsZero() {
+		return
+	}
+	h.k.RefreshIntermediaryDelegationAmounts(ctx)
+}
```

Comment:
We can't backport this PR due to this change. We do need to get the export genesis changes onto v7.x though. Eugen can you make a PR to v7.x with the import / export genesis changes?

Reply:
Actually I'll just have the bot open that PR, and we just comment out these lines